docs/tdvt.md

```diff
@@ -262,6 +267,12 @@ Exclusions = string.ascii
 
 TestPath = tests/mytests/ #This could point to a local directory relative to your TDVT working directory named 'tests'. Or for a logicaltest it would be 'tests/logical/setup/mytests/setup.\.xml
 
+SmokeTest = True # Setting this attribute will cause the test to be run with the Connection Tests and any other
+                 # Smoke Tests at the beginning of the test run or with the --verify flag.
+
+Enabled = False # Optional. The default is True; setting this to false will cause the test to count as a fail
```

Comment:
Maybe I'm not getting the context, but what is enabled? Can this be under any of the test suite headings?